ncm-openstack/src/main/pan/components/openstack/schema.pan

```diff
@@ -0,0 +1,666 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+
+declaration template components/openstack/schema;
+
+include 'quattor/schema';
+include 'pan/types';
+
+
+function is_consistent_glance_store = {
+    # check store values
+    foreach (key; value; SELF) {
+        foreach (key2; value2; SELF) {
+            if (SELF[key] == SELF[key2] && key != key2) {
+                error(format("entry: %s appears several times within glance_store/stores list", value));
```

Comment:
what is the point of this check? if the values are lists with same elements in different order, is that allowed?

Reply:
Or could this be optimised by making glance_store/stores a dict instead?